### PR TITLE
Feature/open close dashboard

### PR DESCRIPTION
**Pull request by MaciejReimann**
1.Add open / close dashboard 
2.Organize desktop layout

Current state of desktop UI: 

Screenshot 2022-03-09 at 19 47 23

### CODE REVIEW

File: src/others/components/MultiTab.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import { styled } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+
+interface MultiTabProps {
+  labels: string[];
+}
+
+// TODO: This one still needs styling
```
Comment (MaciejReimann) [inline src/others/components/MultiTab.tsx:11]: Actually, this one became so complicated when customised, that I'd go with
a) building a custom, simple one
b) take the simplest MUI tabs

**Review by Blightwidow — APPROVED** [review]
One small thing that is not really blocking

File: src/others/components/Dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  children?: ReactNode;
+}
+
+const openDashboardWidth = "300px";
+
+export const Dashboard = ({ children }: DashboardProps) => {
+  const { isOpen } = useDashboardContext();
+
+  return (
+    <Box sx={{ height: "100%", width: isOpen ? openDashboardWidth : 0 }}>
```
Comment (Blightwidow) [inline src/others/components/Dashboard.tsx:17]: Add a small `transition: width .3s ease-in-out` would make it sexier ?

File: src/others/components/dashboard-context.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { ReactNode, useState, useContext, createContext } from "react";
+
+interface DashboardState {
+  isOpen: boolean;
+  toggle: () => void;
+}
+
+export const DashboardContext = createContext<DashboardState | null>(null);
```
[the remark this answers is not in the document]
Comment (MaciejReimann) [reply src/others/components/dashboard-context.tsx:8]: I am not sure if I understand your doubt, let me clarify my intentions. 
DashboardContext should store only what it currently stores, i.e. `isOpen`. 
For filtering purposes, we'll have another context, e.g. FilteredCitiesContext and FilteredCategoriesContext, with setters in respective select components, and getters within the Dashboard. 

That was my idea, at least.

[the remark this answers is not in the document]
Comment (MaciejReimann) [reply src/others/components/dashboard-context.tsx:8]: Let me know what you thing, please.

[the remark this answers is not in the document]
Comment (MaciejReimann) [reply src/others/components/dashboard-context.tsx:8]: Ok, I get your point, I guess it's a naming issue. Dashboard is the whole thing, what I meant is OpenMenu, actually. I am renaming it.

**Pull request merged**